src/core/emergency-action.h

```diff
@@ -28,16 +28,22 @@ typedef enum EmergencyAction {
 } EmergencyAction;
 
 typedef enum EmergencyActionFlags {
-        EMERGENCY_ACTION_IS_WATCHDOG = 1 << 0,
-        EMERGENCY_ACTION_WARN        = 1 << 1,
+        EMERGENCY_ACTION_IS_WATCHDOG = 1 << 0, /* this action triggered by a watchdog or other kind of timeout */
+        EMERGENCY_ACTION_WARN        = 1 << 1, /* log at LOG_WARNING + write to system console */
+        EMERGENCY_ACTION_SLEEP_5S    = 1 << 2, /* wait 5s before executing action; only honoured together with EMERGENCY_ACTION_WARN */
+        _EMERGENCY_ACTION_FLAGS_MAX  = (1 << 3) - 1,
```

Comment:
Not a big matter, but for flags, I'd like to name `_ALL`, rather than `_MAX`.

Reply:
I actually looked across our tree, and some call it _ALL, others _MAX...

i opted for _MAX because that's using in public apis, and hence hard to change those, and probably what we should settle on...

Reply:
Even in the public header, we can use _ALL and keep _MAX for compatibility.